test/integration/api/users/users.list.spec.ts

```diff
@@ -37,6 +37,17 @@ describe("Test GET '/api/users'", () => {
         });
     });
 
+    it(`Users count equal to 2, filtering by the "groupAdminInner" group`, () => {
+      return request(apiService.server)
+        .get(endpoint)
+        .set(apiHelper.getHeaders(apiHelper.superAdminToken))
+        .query({ query: { group: apiHelper.groupAdminInner.id } })
+        .expect(200)
+        .expect((res: any) => {
+          testListCountsAndIds(res, [apiHelper.superAdmin.id, apiHelper.adminInner.id]);
+        });
+    });
```

Comment:
tuo paciu siulau pasirasyt koki nors gudresni query pvz `group: {$in: [...]}` ir ar grazina unikalius userius. Nes dabar padengia tik vieno `number` filtra